opencraft/settings.py

```diff
@@ -300,6 +300,16 @@
 # configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
 OPENEDX_RELEASE_STABLE_REF = env('OPENEDX_RELEASE_STABLE_REF', default='named-release/dogwood.rc')
 
+# The edx-platform repository used by default for production instances
+STABLE_EDX_PLATFORM_REPO_URL = env(
```

Comment:
@jbzdak The reason we have `DEFAULT_FORK` instead of a URL is that it is used by the PR watcher.  The notion of a "fork" and the "organization/repo" syntax are Github-specific, and only needed for the PR watcher.  To clean this up, I would separate the settings for the PR watcher better, and prefix them with `PR_WATCHER`, and write the rest of the code to be completely Github-agnostic.  I thought about cleaning this up now, but figured it is more important to get this out quickly, but I consciously made the decision to use the full  URL instead of the fork name here.